modules/aws-ec2-instance/main.tf

```diff
@@ -24,6 +24,7 @@ resource "aws_instance" "tamr-instance" {
 
 resource "aws_network_interface" "tamr-instance-network" {
   subnet_id       = var.subnet_id
+  private_ips = var.private_ips
```

Comment:
Formatting is off. Run terraform fmt to correct. It has a recursive flag so you can run it at the root of the repo to ensure everything is formatted correctly.